docs/tutorials/javascript/send-payments/create-trust-line-send-currency.md

```diff
@@ -6,805 +6,447 @@ seo:
 labels:
   - Cross-Currency
   - Payments
-  - Quickstart
   - Tokens
 
 ---
 # Create Trust Line and Send Currency Using JavaScript
 
 This example shows how to:
 
-1. Configure accounts to allow transfer of funds to third party accounts.
-2. Set a currency type for transactions.
-3. Create a trust line between the standby account and the operational account.
-4. Send issued currency between accounts.
-5. Display account balances for all currencies.
+1. Create a trust line between the standby account and the operational account.
```

Comment:
The UI no longer mentions Standby and Operational accounts, we should update this in the steps.